example/scalalib/module/2-custom-tasks/build.mill

```diff
@@ -61,7 +61,7 @@ object `package` extends RootModule with ScalaModule {
 // digraph G {
 //   rankdir=LR
 //   node [shape=box width=0 height=0]
-//   ivyDeps -> generatedSources -> compile -> "..."
```

Comment:
is this change to `scalalib` intentional?

Reply:
😂 sorry by mistake (haven't crosschecked)